src/chrome/content/rules/centrum.sk.xml

```diff
@@ -2,40 +2,44 @@
 	For other Centrum.cz coverage, see centrum.cz.xml.
 
 	Nonfunctional subdomains:
-		- (www.)centrum.sk
-		- aktualne.centrum.sk
-		- auto.centrum.sk
 		- automix.centrum.sk
-		- hokej.centrum.sk
 		- icqsearch.centrum.sk
 		- katalog.centrum.sk
 		- lepsiebyvanie.centrum.sk
 		- pobox.centrum.sk
-		- pocasie.centrum.sk
 		- pohladnice.centrum.sk
 		- realitymix.centrum.sk
-		- recepty.centrum.sk
-		- redir.centrum.sk
 		- search.centrum.sk
-		- slovniky.centrum.sk
-		- sutaze.centrum.sk
-		- tv.centrum.sk
-		- zena.centrum.sk
 
 	Mismatched:
 		- (www.)sw.centrum.sk
 		- lite.mail.centrum.sk
 		- m.pmail.centrum.sk
-		- mail.centrum.sk
 		- mail08.centrum.sk
 		- my.centrum.sk
 -->
 <ruleset name="Centrum.sk">
+	<target host="centrum.sk" />
+	<target host="www.centrum.sk" />
+	<target host="aktualne.centrum.sk" />
+	<target host="auto.centrum.sk" />
 	<target host="heslo.centrum.sk" />
+	<target host="hokej.centrum.sk" />
+	<target host="mail.centrum.sk" />
+	<target host="pocasie.centrum.sk" />
 	<target host="preg.centrum.sk" />
 	<target host="puser.centrum.sk" />
+	<target host="recepty.centrum.sk" />
+	<target host="redir.centrum.sk" />
 	<target host="reg.centrum.sk" />
+	<target host="slovniky.centrum.sk" />
+	<target host="sutaze.centrum.sk" />
+	<target host="tv.centrum.sk" />
 	<target host="user.centrum.sk" />
+	<target host="zena.centrum.sk" />
+
+	<rule from="^http://centrum\.sk/"
+		to="https://www.centrum.sk/" />
```

Comment:
http://centrum.sk redirects to http://www.centrum.sk

centrum.sk doesn't support HTTPS

www.centrum.sk supports HTTPS